<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAuxiliarTables</class>
 <widget class="QDialog" name="frmAuxiliarTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auxiliar tables</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/configure.png</normaloff>:/xulpymoney/configure.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Auxiliar tables</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabConceptos">
        <attribute name="title">
         <string>Xulpymoney concepts</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="mqtwObjects" name="mqtwConcepts" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionExpensesAdd">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Add a expense concept</string>
   </property>
   <property name="toolTip">
    <string>Add a expense concept</string>
   </property>
  </action>
  <action name="actionConceptDelete">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/eventdelete.png</normaloff>:/xulpymoney/eventdelete.png</iconset>
   </property>
   <property name="text">
    <string>Delete concept</string>
   </property>
   <property name="toolTip">
    <string>Delete concept</string>
   </property>
  </action>
  <action name="actionIncomesAdd">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Add an income concept</string>
   </property>
   <property name="toolTip">
    <string>Add an income concept</string>
   </property>
  </action>
  <action name="actionChangeName">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Change name</string>
   </property>
   <property name="toolTip">
    <string>Change name</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
